src/components/shared/node_count_selector.js

```diff
@@ -17,6 +17,11 @@ import styled from '@emotion/styled';
 // scaling - The current value of the input field[s].
 //
 
+const DEFAULT_VALUE_CONSTRAINTS = {
+  min: 1,
+  max: 999,
+};
+
```

Comment:
I really think (and I think Adrian will agree) that this should be in a constants file.
We are spreading constants in a lot of components...

Reply:
Ok lets talk in happa2020.. I like having related things close to each other. If possible in the same file, if not in the same folder. Somehow I don't think a big constants file with constants related to all possible components in happa will help us, because then you have to go back and forth between things and it's another context switch. 

But if you guys are strongly in favour of centralising things like that in one place, then I am of course happy to disagree and commit!

Reply:
ok, let's talk about it quickly tomorrow, and decide what is best for us